test/acceptance/features/steps/openshift.py

```diff
@@ -146,7 +146,7 @@ def search_resource_in_namespace(self, resource_plural, name_pattern, namespace)
 
     def is_resource_in(self, resource_type):
         output, exit_code = self.cmd.run(f'{ctx.cli} get {resource_type}')
-        return exit_code == 0
+        return (output, exit_code)
```

Comment:
we changed here fully return values, so `is_resource_in` is not a proper name anymore.. btw, why do we need to change the return value?

Reply:
this function checks if the resource type exists in the cluster. Here we want to make sure if a specific resource exists in the cluster irrespective of the namespace. I suppose we can use resource name directly here without changing the return value as well.